ChoukaBaraGame/project.godot

```diff
@@ -68,6 +68,7 @@ GameUtility="*res://Scripts/Game/GameUtility.gd"
 [debug]
 
 gdscript/completion/autocomplete_setters_and_getters=true
+gdscript/warnings/unused_class_variable=true
```

Comment:
Is this tell me that there are used class variables in your code???

Reply:
This is part of the Lint to keep clean code. it's for everyone.

Reply:
So I assume this has already been addressed in the previous comments?